packages/common/src/view-model-maps/rows/index.js

```diff
@@ -8,6 +8,19 @@ exports.formatRows = (rows, caseData) => {
 	return displayRows.map((row) => createRow(row.keyText, row.valueText));
 };
 
+/**
+ * @param {import('./def').Rows} rows
+ * @param {import("appeals-service-api").Api.AppealCaseWithAppellant} questionnaireData
+ */
+exports.formatQuestionnaireRows = (rows, questionnaireData) => {
+	const displayRows = rows.filter(
+		({ condition }) =>
+			condition(questionnaireData) !== undefined && condition(questionnaireData) !== null
+	);
```

Comment:
```suggestion
	const displayRows = rows.filter(
		({ condition }) => !!condition(questionnaireData)
	);
```

Reply:
I tested this and it seems to remove the questions that have been answered 'No' as I was going on the basis that no would be false.
Generally the boolean questions are Yes/ No radio button answers. I could be wrong but wouldnt this mean Yes = true and No = false? As opposed to the question has been answered = true?
With this in mind the implementation I have filters out only undefined or null values as we would want to keep the false falsy values rather than all falsy values

Reply:
ah yeah fair shout, I hadn't considered that